site/docs/v1/tech/premium-features/scim/okta.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: doc
+title: Okta SCIM Client
+description: Configure Okta as a SCIM client
+navcategory: premium
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+This is a basic example to get started configuring Okta as a SCIM client and establish a connection between FusionAuth and Okta for provisioning purposes.
+
+For complete documentation, please refer to the Okta provisioning documentation.
```

Comment:
Is there a link we can offer folks?

Reply:
Not really, their doc is a loop of endless links. I ended up using 3rd party doc to figure it out. I could can add a link to the root of their doc and let them search.

Reply:
🤯